packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -66,8 +66,8 @@ class VideoPlayer {
     // Allows Safari iOS to play the video inline
     _videoElement.setAttribute('playsinline', 'true');
 
-    // Set autoplay to false since most browsers won't autoplay a video unless it is muted
-    _videoElement.setAttribute('autoplay', 'false');
+    // Ensure autoplay is not set through the DOM!
```

Comment:
Does this mean we should also remove line 63, or is that doing something different? I'm not clear on what exactly the HTML wrapper class properties do.

Reply:
Let me vanilla-JS this bit, I'm not sure if the DOM attribute is synced with the internal state of the element.

Reply:
At least in chrome, they're kept in sync (I'm going to assume this is standard behavior), so we could do either:

<img width="276" alt="Screenshot 2023-09-21 at 11 39 42 AM" src="https://github.com/flutter/packages/assets/1255594/23cbd069-ba6d-4585-9b59-a673aeb8b0b6">

Reply:
I removed the `removeAttribute` call, which is redundant with line 63. Left the check in the tests, which should pass anyway. I also documented why we need to do setAttribute on the "playsinline" property.